<launch>
	<group ns="buoy">
		<node name="monitor_battery_voltage_node" pkg="monitor" type="monitor_battery_voltage" clear_params="true" output="screen">
			<rosparam command="load" file="$(find monitor)/config/monitor_battery_voltage.yaml" />		
		</node>
	</group>

	<node pkg="master_discovery_fkie" type="master_discovery" name="master_discovery_node" output="screen">
		<param name="_mcast_group" value="224.0.0.1"/>
	</node>
</launch>
